mne/viz/epochs.py

```diff
@@ -95,6 +98,8 @@ def plot_epochs_image(epochs, picks=None, sigma=0., vmin=None,
     scale_vmin = True if vmin is None else False
     scale_vmax = True if vmax is None else False
     vmin, vmax = _setup_vmin_vmax(data, vmin, vmax)
+    if logscale:
+        data = np.log2(data)
```

Comment:
I think you can avoid making a copy of the data using `np.log2(data, out=data)`. Otherwise you are storing two huge arrays in memory